src/main/java/emissary/command/HelpCommand.java

```diff
@@ -36,20 +36,24 @@ public String getCommandName() {
     }
 
     @Override
-    public void run(JCommander jc) {
+    public void run(final JCommander jc) {
         setup();
-        if (subcommands.size() == 0) {
+        if (subcommands.isEmpty()) {
             dumpCommands(jc);
         } else if (subcommands.size() > 1) {
             LOG.error("You can only see help for 1 command at a time");
             dumpCommands(jc);
         } else {
-            String subcommand = getSubcommand();
-            LOG.info("Detailed help for: " + subcommand);
+            final String subcommand = getSubcommand();
+            LOG.info("Detailed help for: {}", subcommand);
             try {
-                jc.usage(subcommand);
-            } catch (ParameterException e) {
-                LOG.error("ERROR: invalid command name: " + subcommand);
+                // TODO: Once jCommander 1.83 or higher is released, swap back to jc#usage(subcommand)...
```

Comment:
@dev-mlb mentioned that this flip-flop was a thing but he wasn't sure when 1.83 would actually get released. Is it worth waiting? The other updates in the file seem fine.

Reply:
Just circling back on this, 1.83 still hasn't been released yet and there aren't any signs of it coming any time soon.